.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
-    - "5.4"
```

Comment:
In branch dev-master of pomm are supported php >=5.5.9.
In composer.json when I fork project, it's dev-master referenced in composer.json.
Are you change this ?